src/test.ts

```diff
@@ -0,0 +1,39 @@
+import { create } from './core/player-factory';
```

Comment:
find a better way to use a test template. Maybe provide a template path to haste-preset-playable CLI